Source/WebKit/UIProcess/WebProcessPool.cpp

```diff
@@ -567,6 +567,12 @@ void WebProcessPool::createGPUProcessConnection(WebProcessProxy& webProcessProxy
 
     ensureProtectedGPUProcess()->createGPUProcessConnection(webProcessProxy, WTFMove(connectionIdentifier), WTFMove(parameters));
 }
+
+void WebProcessPool::resetGPUProcess()
```

Comment:
Unclear why we need a new function on the WebProcessPool. Why can't the caller just call terminate() on the GPUProcessProxy directly?